app/components/rdf-form-fields/property-group-selector.js

```diff
@@ -27,7 +27,7 @@ export default class PropertyGroupSelector extends InputFieldComponent {
   loadOptions() {
     const sourceGraph = this.args.graphs.sourceGraph;
     const propertyGroup = new namedNode(
-      'http://lblod.data.gift/vocabularies/forms/PropertyGroup'
+      'http://lblod.data.gift/vocabularies/forms/Section'
     );
```

Comment:
This keeps annoying me haha I know there is a ticket for it 👌🏻 
Don't forget to also rename the component in that ticket :)